library/dnet_common.c

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2008+ Evgeniy Polyakov <zbr@ioremap.net>
+ * Copyright 2015+ Yandex
```

Comment:
This is a rather small feature compared to the whole file, just drop this.

I'm about to remove copyright notices and add authors file instead anyway. Authors have copyrights on corresponding code changes, which is observable via git tools